packages/shared/src/api/postsApi.ts

```diff
@@ -117,12 +121,36 @@ export const sendPost = async ({
     return;
   }
 
+  const kindData = (): ub.KindData => {
```

Comment:
Could this be extracted into a `toWrit` helper or something? We prob need some similar things for `editPost`, and maybe in other places.